src/BenchmarkDotNet/Running/BenchmarkRunnerDirty.cs

```diff
@@ -16,18 +20,22 @@ namespace BenchmarkDotNet.Running
     // so it can ignore the version mismatch ;)
     public static class BenchmarkRunner
     {
+        private static readonly IUserInteraction userInteraction = new UserInteraction();
+        private static readonly List<Type> types = new List<Type>();
```

Comment:
this should be a local variable of the `RunWithDirtyAssemblyResolveHelper` method

Reply:
yes, makes sense.